components/Home/Bonding/TokenManagement/hooks/useWsolWithdraw.jsx

```diff
@@ -0,0 +1,200 @@
+import { useCallback } from 'react';
+import { Program } from '@coral-xyz/anchor';
+import idl from 'common-util/AbiAndAddresses/liquidityLockbox.json';
+import { DecimalUtil, Percentage } from '@orca-so/common-sdk';
+import { decreaseLiquidityQuoteByLiquidityWithParams } from '@orca-so/whirlpools-sdk';
+import {
+  AccountLayout,
+  getAssociatedTokenAddress,
+  TOKEN_PROGRAM_ID,
+} from '@solana/spl-token';
+import Decimal from 'decimal.js';
+import { notifyError, notifySuccess } from '@autonolas/frontend-library';
+
+import { useSvmConnectivity } from 'common-util/hooks/useSvmConnectivity';
+import { useGetOrCreateAssociatedTokenAccount } from './useGetOrCreateAssociatedTokenAccount';
+import { useWhirlpool } from './useWhirlpool';
+import {
+  BRIDGED_TOKEN_MINT,
+  FEE_COLLECTOR_TOKEN_OWNER_ACCOUNT_A,
+  FEE_COLLECTOR_TOKEN_OWNER_ACCOUNT_B,
+  LOCKBOX,
+  ORCA,
+  PDA_POSITION_ACCOUNT,
+  POSITION,
+  PROGRAM_ID,
+  TICK_ARRAY_LOWER,
+  TICK_ARRAY_UPPER,
+  TOKEN_VAULT_A,
+  TOKEN_VAULT_B,
+  WHIRLPOOL,
+  POSITION_MINT,
+  tickLowerIndex,
+  tickUpperIndex,
+  CONNECT_SVM_WALLET,
+} from '../constants';
+
+const TOKEN_MINT_ERROR = 'You do not have the correct token account, please try again.';
```

Comment:
This doesn't read right, unsure what the error should be though, either:
- `You do not have the correct token amount, please try again`
- `You do not have the required tokens in your account, please try again`